src/clincoded/static/libs/render_variant_title.js

```diff
@@ -14,6 +14,8 @@ export function renderVariantTitle(variant, stringOnly) {
     if (variant) {
         if (variant.clinvarVariantId && variant.clinvarVariantTitle) {
             variantTitle = variant.clinvarVariantTitle;
+        } else if (variant.maneTranscriptTitle) {
```

Comment:
It wouldn't hurt to update the function declaration comment block (to include this new option).